Documentation/upgrade.md

```diff
@@ -68,22 +89,28 @@ spec:
 ```
 
 ## Health Verification
-Before we begin the upgrade process, let's first review some ways that you can verify the health of your cluster, ensuring that the upgrade is going smoothly after each step.
-Most of the health verification checks for your cluster during the upgrade process can be performed with the Rook toolbox.
-For more information about how to run the toolbox, please visit the [Rook toolbox readme](./toolbox.md#running-the-toolbox-in-kubernetes).
+Before we begin the upgrade process, let's first review some ways that you can verify the health of
+your cluster, ensuring that the upgrade is going smoothly after each step. Most of the health
+verification checks for your cluster during the upgrade process can be performed with the Rook
+toolbox. For more information about how to run the toolbox, please visit the
+[Rook toolbox readme](./toolbox.md#running-the-toolbox-in-kubernetes).
 
 ### Pods all Running
-In a healthy Rook cluster, the operator, the agents and all Rook namespace pods should be in the `Running` state and have few, if any, pod restarts.
-To verify this, run the following commands:
+In a healthy Rook cluster, the operator, the agents and all Rook namespace pods should be in the
+`Running` state and have few, if any, pod restarts. To verify this, run the following commands:
 ```bash
 kubectl -n rook-system get pods
```

Comment:
the default namespaces in the sample commands should be `rook-ceph-system` and `rook-ceph`